### PR TITLE
Update based on the treecounter type

### PR DESCRIPTION
**Pull request by mahmed0715**
@sagararyal

### CODE REVIEW

**Comment by norbertschuler** [comment]
Is this PR fixing any issue? It's difficult to see which changes are because of reformatted code and what is new or changed code.

**Comment by mahmed0715** [comment]
this is a hotfix, i made two pr's mistakenly, did you merge master to develop after hotfix? @sagararyal @norbertschuler

**Comment by norbertschuler** [comment]
> this is a hotfix, i made two pr's mistakenly, did you merge master to develop after hotfix? @sagararyal @norbertschuler

Sure, if this PR get merged here with GitHub, I would merge master back into develop. Or @sagararyal or me can use SourceTree with the GitFlow to finish the Hotfix, which would merge it into master and develop at once.

**Comment by mahmed0715** [comment]
I guess my other PR(that is already merged to master), may be its not get merged to develop yet

**Comment by norbertschuler** [comment]
> I guess my other PR(that is already merged to master), may be its not get merged to develop yet

No it was just closed without merging.

File: app/containers/DonateTrees/index.js [hunk 1]
```diff
@@ -44,8 +44,10 @@ class DonationTreesContainer extends PureComponent {
             _suggestions.data.length &&
             _suggestions.data[0].slug == match.params.slug
           ) {
-            supportTreecounterAction({
+            console.log('support treecounter suggestions', _suggestions.data[0]);
```
Comment (norbertschuler) [inline app/containers/DonateTrees/index.js:47]: please replace console.log with debug()

File: app/locales/en/publicTreecounterlabels.json [hunk 2]
```diff
@@ -6,6 +6,7 @@
   "planted_trees": "Planted Trees",
   "target": "Target",
   "support_trees_to": "Support trees to {{user}}",
```
Comment (norbertschuler) [inline app/locales/en/publicTreecounterlabels.json:8]: Is this proper English? I would rather say `Support {{user}} with trees`. Currently it looks like this on production:

Bildschirmfoto 2020-04-04 um 11 33 09

File: app/locales/en/publicTreecounterlabels.json [hunk 3]
```diff
@@ -6,6 +6,7 @@
...
+  "give_trees_to_company": "Give trees with {{companyName}}",
```
Comment (norbertschuler) [inline app/locales/en/publicTreecounterlabels.json:9]: Also I am wondering if this is a nice spelling? What should it mean?

**Pull request merged**